rmgpy/rmg/reactTest.py

```diff
@@ -116,10 +121,11 @@ def testReactAll(self):
                 ]
 
         n = len(spcs)
-        reaction_list, spc_tuples = react_all(spcs, n, np.ones(n), np.ones([n, n]), np.ones([n, n, n]), procnum)
+        reaction_list, spc_tuples = react_all(spcs, n, np.ones([n, num_families]), np.ones([n, n, num_families]),
+                                              np.ones([n, n, n, num_families]), procnum)
         self.assertIsNotNone(reaction_list)
-        self.assertEqual(len(reaction_list), 34)
-        self.assertEqual(len(spc_tuples), 34)
+        self.assertEqual(len(reaction_list), 27)
+        self.assertEqual(len(spc_tuples), 27)
```

Comment:
@mliu49 can you check this before you merge?

Reply:
Confirmed that this is ok.